samples/CommunityToolkit.Maui.Sample/Views/Popups/ApplyToDerivedTypesPopup.xaml

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<mct:Popup xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:mct="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="CommunityToolkit.Maui.Sample.Views.Popups.ApplyToDerivedTypesPopup">
+    <mct:Popup.Resources>
```

Comment:
I wonder if there is some value in adding xml comments to inform the reader what each style based scenario is covering as we want to use this as learning material. It could be something like:

```xaml
<!-- This popup demonstrates how the ApplyToDerivedTypes property can allow for Popup implementations that inherit from Popup to still use a common Style definition. -->
```

Reply:
I think it would be easier to understand if there was an explanation in the comments for each style.
Is it possible for you to write comments for each style?

Reply:
Yep I would be happy to take this on 👍

Reply:
I've opted to add XML docs instead. As it will hopefully be more discoverable.

Reply:
@bijington , Thank you for your willingness to accept. 
Thank you for your help.